File: flutter_fimber/example/ios/Classes/FlutterFimberPlugin.m
#import "FlutterFimberPlugin.h"
#import <flutter_fimber/flutter_fimber-Swift.h>

@implementation FlutterFimberPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  [SwiftFlutterFimberPlugin registerWithRegistrar:registrar];
}
@end

File: flutter_fimber/macos/Classes/FlutterFimberPlugin.swift
import Cocoa
import FlutterMacOS

public class FlutterFimberPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    let channel = FlutterMethodChannel(name: "flutter_fimber", binaryMessenger: registrar.messenger)
    let instance = FlutterFimberPlugin()
    registrar.addMethodCallDelegate(instance, channel: channel)
  }

  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
    switch call.method {
    case "getPlatformVersion":
      result("macOS " + ProcessInfo.processInfo.operatingSystemVersionString)
    default:
      result(FlutterMethodNotImplemented)
    }
  }
}
